html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html-forms</title>
+  </head>
+  <body>
+    <form action="index.html" method="post">
+      <h1>Coding Form!</h1>
+      <label for="email">Email:</label>
+      <input type="email" id="email" name="email" />
+
+      <p>Why is coding cool?</p>
```

Comment:
This should be a `label` that will need to be linked to your `select` via a `for` attribute